### PR TITLE
Pass at clarifying bridge instructions

### PR DESCRIPTION
**Pull request by baudtack**

### CODE REVIEW

File: getting-started.udon [hunk 1]
```diff
@@ -94,7 +94,15 @@ seed is also useful to perform everyday operations (such as boot your ship!).
 
 Select the appropriate option and click `Continue ->`.
 
-#### Step 4: (optional) Accept Your Transfer
+If you already have a point, (for example, if you are trying to boot a ship
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
If you already have a point -- for example, if you are trying to boot a ship```

File: getting-started.udon [hunk 2]
```diff
@@ -94,7 +94,15 @@ seed is also useful to perform everyday operations (such as boot your ship!).
...
+from a pre-etherium era network) you should see a list of points associated with your
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
from a pre-Ethereum-era network -- you should see a list of points associated with your
```

File: getting-started.udon [hunk 3]
```diff
@@ -94,7 +94,15 @@ seed is also useful to perform everyday operations (such as boot your ship!).
...
+keys. If you are trying to boot a planet for the first time, click on the
+details link for that point. At the bottom on the details page there is
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
`Details` link for that point. At the bottom on the Details page, there is
```

File: getting-started.udon [hunk 4]
```diff
@@ -94,7 +94,15 @@ seed is also useful to perform everyday operations (such as boot your ship!).
...
+a section called `Urbit Networking` if this section is filled out, proceed to step 6.
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
a section called `Urbit Networking`. If the fields in this section are filled out, proceed to step 6.
```

File: getting-started.udon [hunk 5]
```diff
@@ -94,7 +94,15 @@ seed is also useful to perform everyday operations (such as boot your ship!).
...
+
+If you do not have a point and are getting one sent to you, proceed to step 4.
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
If you do not yet have a point and are getting one sent to you, proceed to step 4.
```

File: getting-started.udon [hunk 6]
```diff
@@ -117,6 +125,9 @@ network seed and derive a new keyfile. See our
 [HD Wallet Spec](https://github.com/urbit/proposals/blob/master/8.udon)
 for more information.
 
+It should be noted that this is an event on the etherium network and will cost
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
It should be noted that this is an event on the Ethereum network and will cost
```

File: getting-started.udon [hunk 7]
```diff
@@ -117,6 +125,9 @@ network seed and derive a new keyfile. See our
...
+(generally) a trivial amount of gas to complete.
```
Comment (rmariani) [inline getting-started.udon]:
```suggestion
 a trivial, but non-zero, amount of [gas](https://github.com/ethereum/wiki/wiki/Design-Rationale#gas-and-fees) to complete.
```

**Comment by vvisigoth** [comment]
@rmariani can you just make these copy edits and push to this branch?

**Comment by rmariani** [comment]
What would "this branch" be? David is PRing from his personal repo

**Comment by ohAitch** [comment]
I think there's a github flag you can set where PR target maintainers can
add commits to the relevant personal repo branch, idk if that's enabled here

**Comment by vvisigoth** [comment]
ah crap, true.

**Comment by rmariani** [comment]
LGTM

**Pull request merged**